src/main/scala/cromwell/engine/db/slick/RunMysql.scala

```diff
@@ -141,8 +141,8 @@ object RunMysql {
     getConfigOpt("properties." + key) orElse getConfigOpt(key) getOrElse ""
 
   private def getConfigOpt(key: String): Option[String] = {
```

Comment:
There is a method for this in ConfigUtil (more specifically in an implicit class that extends Config)
`def getStringOption(key: String): Option[String]`